icons/keras/keras-line.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M124.87 3.13v121.73H3.13V3.13h121.73M48.41 56.42V30.04c0-.35-.15-.72-.38-.98l-2.08-2.23a1.43 1.43 0 0 0-1.02-.44h-7.3c-.37 0-.76.17-1.02.44l-2.13 2.3c-.24.25-.38.63-.38.98v68.08c.03.33.18.66.41.89l2.12 2.12c.25.25.64.41 1 .41h7.25c.36 0 .74-.16 1-.41L48 99.08c.25-.25.41-.64.41-1V75.34l8.29-7.92 22.7 33.6c.24.35.67.58 1.09.58h10.27c.47 0 .92-.28 1.14-.69l1.86-3.61c.21-.41.18-.94-.09-1.32l-26.83-38.9L91.4 32.64c.25-.25.41-.64.41-1v-.46c0-.23-.06-.51-.15-.72l-1.44-3.32c-.2-.45-.66-.76-1.16-.76H78.91c-.36 0-.74.16-1 .41L48.41 56.4m77.45-54.27H2.14v123.72h123.72V2.14Zm-78.4 56.63s.04-.01.07-.04l31.09-31.21c.07-.07.2-.12.29-.12h10.15c.09 0 .2.07.24.16l1.44 3.32c.04.09.07.23.07.33v.46c0 .09-.05.22-.12.29L65.66 56.86c-.07.07-.08.18-.02.26l27.21 39.44c.05.08.06.21.02.29l-1.86 3.61c-.04.08-.15.15-.25.15H80.49c-.09 0-.21-.06-.26-.14L56.97 66.03s-.07-.07-.12-.07c-.04 0-.07.01-.1.04l-9.21 8.79c-.07.06-.12.19-.12.29v23c0 .09-.05.22-.12.29l-2.12 2.12c-.07.07-.2.12-.29.12h-7.25c-.09 0-.22-.05-.29-.12l-2.12-2.12a.467.467 0 0 1-.12-.29V30.11c0-.09.05-.23.11-.29l2.13-2.3c.06-.07.19-.12.29-.12h7.3c.09 0 .22.06.29.12l2.08 2.23c.06.07.12.2.12.29v28.63c0 .06.02.09.05.09Z"/></svg>
```

Comment:
Can we remove the padding on this icon, so it fills the entire viewbox? The same goes for original and plain versions.

Reply:
I removed the padding and noted it for the future.

Reply:
@ConX Excellent! We should definitely update the wiki as well. The reason it's not there yet is because having 0 padding was a fairly recent decision made because of a suggestion from a third-party developer :)